source/_docs/custom-certificates.md

```diff
@@ -108,6 +108,13 @@ Please allow two business days to get a CSR and load the certificate.
 
 Before your custom certificate expires, Pantheon will open a ticket with your team with a new CSR. You can send that CSR to the Certificate Authority to generate new certificates (as described above for <a href="#option-2-manually-managed-custom-certificates" data-proofer-ignore>bringing a custom certificate</a>).
 
+To request a new CSR ahead of time, please [contact support](/docs/support/) with the following details:
```

Comment:
Thanks @alexfornuto. cc @ttrowell 
How about making this a section that is linkable? Maybe in FAQ?

*What is the process to replace an existing certificate?*

We will provide you with a CSR 45 days prior to your certificate expiration so you can obtain a new certificate. If your use case requires frequently replacing a certificate well ahead of expiration to, for example, add additional domains, then you may replace an existing certificate without needing a new CSR from Pantheon.

To replace a certificate well in advance of expiration, please contact support.....

Reply:
@ari-gold This is an addendum to an existing FAQ section named "How do I renew my custom certificate". Do you think it needs its own header, or do you agree that it's a good edition to the existing section?

Reply:
@alexfornuto Good call. Yes. looks like it could be combined to that existing FAQ.